<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1335</width>
    <height>802</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
  background-color: #19232D;
  border: 0px solid #455364;
  padding: 0px;
  color: white;
  selection-background-color: #346792;
  selection-color: #E0E1E3;
	font-family : Ubuntu;
font-size: 18px;
font-weight: bold;
}

QWidget:disabled {
  background-color: #19232D;
  color: #9DA9B5;
  selection-background-color: #26486B;
  selection-color: #9DA9B5;
}

QWidget::item:selected {
  background-color: #176B87;
}

QWidget::item:hover:!selected {
  background-color: rgba(23, 107, 135,50%);
}

QMainWindow::separator {
  background-color: #455364;
  border: 0px solid #19232D;
  spacing: 0px;
  padding: 2px;
}

QMainWindow::separator:hover {
  background-color: #60798B;
  border: 0px solid #1A72BB;
}

/*-------------------------------------------------------------------------------------------*/

QCheckBox {
  background-color: rgb(0,0,0,0);
  color: white;
  spacing: 4px;
  outline: none;
  padding-top: 2px;

}


/*-------------------------------------------------------------------------------------------*/
QScrollBar:horizontal {
  height: 16px;
  margin: 2px 16px 2px 16px;
  border: 1px solid white;
  border-radius: 4px;
  background-color: rgb(25, 35, 45);
}

QScrollBar::handle:horizontal {
  background-color: #176B87;
  border: 1px solid #C9CDD0;
  border-radius: 4px;
  min-width: 8px;
}

QScrollBar::handle:horizontal:hover {
  background-color: rgba(23, 107, 135,80%);
  border:#4169E1;
  border-radius: 4px;
  min-width: 8px;
}

QScrollBar::handle:horizontal:focus {
  border: 1px solid red;
}
QScrollBar:vertical {
  background-color: rgb(25, 35, 45);
  width: 16px;
  margin: 16px 2px 16px 2px;
  border: 1px solid white;
  border-radius: 4px;
}


QScrollBar::handle:vertical {
  background-color: #176B87;
  border: 1px solid #C9CDD0;
  min-height: 8px;
  border-radius: 4px;
}

QScrollBar::handle:vertical:hover {
  background-color: rgba(23, 107, 135,80%);
  border: #9FCBFF;
  border-radius: 4px;
  min-height: 8px;
}

QScrollBar::handle:vertical:focus {
  border: 1px solid #73C7FF;
}


QScrollBar::add-line:horizontal {
  margin: 0px 0px 0px 0px;
  border-image: url(&quot;qss_icons/light/rc/arrow_right_disabled.png&quot;);
  height: 12px;
  width: 12px;
  subcontrol-position: right;
  subcontrol-origin: margin;
}

QScrollBar::add-line:horizontal:hover, QScrollBar::add-line:horizontal:on {
  border-image: url(&quot;qss_icons/light/rc/arrow_right.png&quot;);
  height: 12px;
  width: 12px;
  subcontrol-position: right;
  subcontrol-origin: margin;
}

QScrollBar::add-line:vertical {
  margin: 3px 0px 3px 0px;
  border-image: url(&quot;qss_icons/light/rc/arrow_down_disabled.png&quot;);
  height: 12px;
  width: 12px;
  subcontrol-position: bottom;
  subcontrol-origin: margin;
}

QScrollBar::add-line:vertical:hover, QScrollBar::add-line:vertical:on {
  border-image: url(&quot;qss_icons/light/rc/arrow_down.png&quot;);
  height: 12px;
  width: 12px;
  subcontrol-position: bottom;
  subcontrol-origin: margin;
}

QScrollBar::sub-line:horizontal {
  margin: 0px 3px 0px 3px;
  border-image: url(&quot;qss_icons/light/rc/arrow_left_disabled.png&quot;);
  height: 12px;
  width: 12px;
  subcontrol-position: left;
  subcontrol-origin: margin;
}

QScrollBar::sub-line:horizontal:hover, QScrollBar::sub-line:horizontal:on {
  border-image: url(&quot;qss_icons/light/rc/arrow_left.png&quot;);
  height: 12px;
  width: 12px;
  subcontrol-position: left;
  subcontrol-origin: margin;
}

QScrollBar::sub-line:vertical {
  margin: 3px 0px 3px 0px;
  border-image: url(&quot;qss_icons/light/rc/arrow_up_disabled.png&quot;);
  height: 12px;
  width: 12px;
  subcontrol-position: top;
  subcontrol-origin: margin;
}

QScrollBar::sub-line:vertical:hover, QScrollBar::sub-line:vertical:on {
  border-image: url(&quot;qss_icons/light/rc/arrow_up.png&quot;);
  height: 12px;
  width: 12px;
  subcontrol-position: top;
  subcontrol-origin: margin;
}

QScrollBar::up-arrow:horizontal, QScrollBar::down-arrow:horizontal {
  background: none;
}

QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
  background: none;
}


QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal {
  background: none;
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
  background: none;
}

/*----------------------------------------------------------------------------------------*/
QPushButton{
 /*border:2px solid #05B8CC;*/
 background-color: #176B87;
padding: 3px 10px;
 color:rgb(255, 255, 255);
 border-radius: 10px;
 font-weight:bold;
  margin: 5px;
}

QPushButton::hover{
 border: 1px solid #176B87;
 background-color: rgba(23, 107, 135,80%)
}
QPushButton:pressed {
 margin:1px 2px;
 font-size: 15px;
}
/*----------------------------------------------------------------------------------------*/
QComboBox{
 border:1px solid #176B87;
border-radius:5px;
background-color: #176B87;
padding: 2px 10px;
color:white;
}



QComboBox:on { /* shift the text when the popup opens */
    padding-top: 3px;
    padding-left: 4px;
}


QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 22px;

    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid; /* just a single line */
    border-top-right-radius: 3px; /* same radius as the QComboBox */
    border-bottom-right-radius: 3px;
}

QComboBox::down-arrow {
    image: url(icons/caret-down copy.svg);
}

QComboBox::down-arrow:on { /* shift the arrow when popup is open */
    top: 1px;
    left: 1px;
}
QComboBox QAbstractItemView {
border:none;
    selection-background-color:#176B87;
}
/*---------------------------------------------------------------------------------------*/
QProgressBar{
	border: 1px solid #05B8CC;
	border-radius:5px;
	text-align:center;
	font-size:18px;
}

QProgressBar::chunk{
	background-color: #05B8CC;
	width: 17px;
	margin: 0.5px;
}
/*----------------------------------------------------------------------------------------*/
QTabBar::tab {
    background-color: #455364;
    border: 3px solid #19232D;
    border-radius: 10px;
    padding: 5px;
    color: #E0E1E3;
	width:150px;
    font-family: Ubuntu;
    font-size: 20px;
    font-weight: bold;
}
QTabBar::tab {
    background-color: #455364;
    border: 3px solid #19232D;
    border-radius: 10px;
    padding: 4px 8px;
    color: #E0E1E3;
    font-family: Ubuntu;
    font-size: 20px;
    font-weight: bold;
}
QTabBar::tab:selected {
    background-color:rgba(69,83,100,60%);
    color: white;
	margin-top: 3px;
	border-bottom: 3px solid white;
}
QPlotWidget{
 background-color:(25, 35, 45, 0.8)
}


QCheckBox{
	font-size: 18px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="upper_frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>7</number>
         </property>
         <property name="topMargin">
          <number>7</number>
         </property>
         <property name="rightMargin">
          <number>7</number>
         </property>
         <property name="bottomMargin">
          <number>7</number>
         </property>
         <item row="0" column="0">
          <widget class="QPushButton" name="close_left_frame">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>open left frame</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>icons/brightness.svg</normaloff>icons/brightness.svg</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QPushButton" name="pushButton_9">
           <property name="text">
            <string>close</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="pushButton_7">
           <property name="text">
            <string>minmize</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="pushButton_8">
           <property name="text">
            <string>maxmize</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,6">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="left_frame">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>500</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <property name="leftMargin">
            <number>7</number>
           </property>
           <property name="topMargin">
            <number>7</number>
           </property>
           <property name="rightMargin">
            <number>7</number>
           </property>
           <property name="bottomMargin">
            <number>7</number>
           </property>
           <item row="6" column="0">
            <widget class="QPushButton" name="IVF_btn">
             <property name="text">
              <string>IVF lab</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QPushButton" name="anesthesia_btn">
             <property name="text">
              <string>Anesthesia and Recovery</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="clinics_btn">
             <property name="text">
              <string>Clinics</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QPushButton" name="blood_btn">
             <property name="text">
              <string>Blood and Andrology </string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="operation_btn">
             <property name="text">
              <string>Operations </string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="main_frame">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QTabWidget" name="main_tabWidget">
             <property name="tabShape">
              <enum>QTabWidget::Rounded</enum>
             </property>
             <property name="currentIndex">
              <number>0</number>
             </property>
             <property name="elideMode">
              <enum>Qt::ElideMiddle</enum>
             </property>
             <property name="usesScrollButtons">
              <bool>true</bool>
             </property>
             <property name="documentMode">
              <bool>true</bool>
             </property>
             <property name="tabsClosable">
              <bool>true</bool>
             </property>
             <property name="movable">
              <bool>true</bool>
             </property>
             <property name="tabBarAutoHide">
              <bool>false</bool>
             </property>
             <widget class="QWidget" name="operation_tab">
              <attribute name="title">
               <string>Operation</string>
              </attribute>
              <layout class="QGridLayout" name="gridLayout_5">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <property name="spacing">
                <number>0</number>
               </property>
               <item row="0" column="0">
                <widget class="QTabWidget" name="operation_tabWidget">
                 <property name="currentIndex">
                  <number>0</number>
                 </property>
                 <property name="documentMode">
                  <bool>true</bool>
                 </property>
                 <property name="tabsClosable">
                  <bool>true</bool>
                 </property>
                 <property name="movable">
                  <bool>true</bool>
                 </property>
                 <property name="tabBarAutoHide">
                  <bool>true</bool>
                 </property>
                 <widget class="QWidget" name="defibrilator_tab">
                  <attribute name="title">
                   <string>Defibrilator</string>
                  </attribute>
                  <layout class="QGridLayout" name="gridLayout_9">
                   <item row="0" column="0">
                    <widget class="QTableWidget" name="defibrilator_table">
                     <property name="styleSheet">
                      <string notr="true">QHeaderView::section{
	border: 1px solid #ccc;
	background-color: rgba(255, 255, 255, 0);
	font-size: 19px;

}

QTableWidget{
	border: none;
	  background-color: #19232D;
	font-size: 18px;
	font-weight: normal;
}
QTableWidget::item{
border: 1px solid #ccc;
}
</string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="0">
                    <layout class="QHBoxLayout" name="horizontalLayout_2">
                     <item>
                      <spacer name="horizontalSpacer_2">
                       <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>40</width>
                         <height>20</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                     <item>
                      <widget class="QPushButton" name="add_defibrilator_btn">
                       <property name="text">
                        <string>Add device</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </widget>
                 <widget class="QWidget" name="diathermy_tab">
                  <attribute name="title">
                   <string>Diathermy</string>
                  </attribute>
                 </widget>
                 <widget class="QWidget" name="electrosurg_tab">
                  <attribute name="title">
                   <string>Electrosurg</string>
                  </attribute>
                 </widget>
                 <widget class="QWidget" name="monitor_tab">
                  <attribute name="title">
                   <string>Monitor</string>
                  </attribute>
                 </widget>
                 <widget class="QWidget" name="suction_tab">
                  <attribute name="title">
                   <string>Suction</string>
                  </attribute>
                 </widget>
                 <widget class="QWidget" name="ultrasound_tab">
                  <attribute name="title">
                   <string>Ultrasound</string>
                  </attribute>
                 </widget>
                 <widget class="QWidget" name="vaporizer_tab">
                  <attribute name="title">
                   <string>Vaporizer</string>
                  </attribute>
                 </widget>
                 <widget class="QWidget" name="ventilator_tab">
                  <attribute name="title">
                   <string>Ventilator</string>
                  </attribute>
                 </widget>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="clinics_tab">
              <attribute name="title">
               <string>Clinics</string>
              </attribute>
              <layout class="QGridLayout" name="gridLayout_6">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <property name="spacing">
                <number>0</number>
               </property>
               <item row="0" column="0">
                <widget class="QTabWidget" name="clinics_tabWidget">
                 <property name="documentMode">
                  <bool>true</bool>
                 </property>
                 <property name="tabsClosable">
                  <bool>true</bool>
                 </property>
                 <property name="movable">
                  <bool>true</bool>
                 </property>
                 <property name="tabBarAutoHide">
                  <bool>true</bool>
                 </property>
                 <widget class="QWidget" name="tab_8">
                  <attribute name="title">
                   <string>Tab 1</string>
                  </attribute>
                 </widget>
                 <widget class="QWidget" name="tab_10">
                  <attribute name="title">
                   <string>Page</string>
                  </attribute>
                 </widget>
                 <widget class="QWidget" name="tab_9">
                  <attribute name="title">
                   <string>Tab 2</string>
                  </attribute>
                 </widget>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="anesthesia_tab">
              <attribute name="title">
               <string>Anesthesia</string>
              </attribute>
              <layout class="QGridLayout" name="gridLayout_7">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <property name="spacing">
                <number>0</number>
               </property>
               <item row="0" column="0">
                <widget class="QTabWidget" name="anesthesia_tabWidget">
                 <property name="documentMode">
                  <bool>true</bool>
                 </property>
                 <property name="tabsClosable">
                  <bool>true</bool>
                 </property>
                 <property name="movable">
                  <bool>true</bool>
                 </property>
                 <property name="tabBarAutoHide">
                  <bool>true</bool>
                 </property>
                 <widget class="QWidget" name="tab_6">
                  <attribute name="title">
                   <string>Tab 1</string>
                  </attribute>
                 </widget>
                 <widget class="QWidget" name="tab_12">
                  <attribute name="title">
                   <string>Page</string>
                  </attribute>
                 </widget>
                 <widget class="QWidget" name="tab_13">
                  <attribute name="title">
                   <string>Page</string>
                  </attribute>
                 </widget>
                 <widget class="QWidget" name="tab_11">
                  <attribute name="title">
                   <string>Tab 2</string>
                  </attribute>
                 </widget>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="blood_tab">
              <attribute name="title">
               <string>Blood lab</string>
              </attribute>
              <layout class="QGridLayout" name="gridLayout_8">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <property name="spacing">
                <number>0</number>
               </property>
               <item row="0" column="0">
                <widget class="QTabWidget" name="blood_tabWidget">
                 <property name="currentIndex">
                  <number>0</number>
                 </property>
                 <property name="elideMode">
                  <enum>Qt::ElideMiddle</enum>
                 </property>
                 <property name="documentMode">
                  <bool>true</bool>
                 </property>
                 <property name="tabsClosable">
                  <bool>true</bool>
                 </property>
                 <property name="movable">
                  <bool>true</bool>
                 </property>
                 <property name="tabBarAutoHide">
                  <bool>true</bool>
                 </property>
                 <widget class="QWidget" name="tab_14">
                  <attribute name="title">
                   <string>Tab 1</string>
                  </attribute>
                 </widget>
                 <widget class="QWidget" name="tab_16">
                  <attribute name="title">
                   <string>Page</string>
                  </attribute>
                 </widget>
                 <widget class="QWidget" name="tab_17">
                  <attribute name="title">
                   <string>Page</string>
                  </attribute>
                 </widget>
                 <widget class="QWidget" name="tab_18">
                  <attribute name="title">
                   <string>Page</string>
                  </attribute>
                 </widget>
                 <widget class="QWidget" name="tab_15">
                  <attribute name="title">
                   <string>Tab 2</string>
                  </attribute>
                 </widget>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="IVF_tab">
              <attribute name="title">
               <string>IVF lab</string>
              </attribute>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
